reqs/management/commands/import_reqs.py

```diff
@@ -111,12 +134,35 @@ def __init__(self, fields):
         self.cache = {}
         self.fields = fields
 
+    def normalize_keywords(self, values):
+        """
+        Executive decisions:
+        +   Keywords cannot contain whitespace at the start or end.
+        +   All whitespace will be converted to a single space.
+        +   Keyword values must be init-caps.
+        +   Use of hyphens as separators requires a single space around the
+            hyphen.
```

Comment:
I'm uncertain about these last two, but I'm cool with using them to clean the data; I don't think they can hurt.